stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampInteractor.kt

```diff
@@ -264,9 +270,180 @@ internal class OnrampInteractor @Inject constructor(
     fun onLinkControllerState(linkState: LinkController.State) {
         _state.value = _state.value.copy(linkControllerState = linkState)
     }
+
+    /**
+     * Starts the checkout flow for a crypto onramp session. The checkout state will be emitted
+     * through the state StateFlow. The coordinator should observe the checkoutState and react accordingly.
+     *
+     * @param onrampSessionId The onramp session identifier.
+     * @param checkoutHandler An async closure that calls your backend to perform a checkout.
+     */
+    suspend fun startCheckout(
+        onrampSessionId: String,
+        checkoutHandler: suspend () -> String
+    ) {
+        cryptoApiRepository.getPlatformSettings()
```

Comment:
Follow-up PR retrieving platform setting when calling configure and reusing it coming soon!